server/src/utils/fight/fightMethods.ts

```diff
@@ -981,10 +1000,13 @@ const activateSuper = (
         // Change pet owner
         pet.master = fighter.id;
         pet.team = fighter.team;
+        // Lower pet initiative
+        pet.initiative -= 1
```

Comment:
Why lower the pet initiative?

Reply:
The idea is to give hypnotised pets an initiative boost, so hypnosis "unleach" them on their master.

I can give them a visual effect like you are saying for the pissed pets, and also do it for treat.